### PR TITLE
[FAB-17971] Upgrade network from 1.4 to 2.2

### PR DESCRIPTION
**Pull request by suryalnvs**
Updated scripts to support upgrade to 2.2 from 1.4

Signed-off-by: Surya Lanka <suryalnvs@gmail.com>

### CODE REVIEW

File: tools/operator/main.go [hunk 1]
```diff
@@ -79,6 +79,12 @@ func doAction(action, env, kubeConfigPath, inputFilePath string) error {
 			logger.ERROR("Failed to upgrade network")
 			return err
 		}
+	case "upgradeDB":
+		err = launcher.Launcher("upgradeDB", env, kubeConfigPath, inputPath)
+		if err != nil {
+			logger.ERROR("Failed to upgrade network")
```
Comment (scottz64) [inline tools/operator/main.go]: Failed to upgrade database

File: regression/upgrade/upgrade1.4to2.2.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+cd $OperatorDir
+echo "Upgrading fabric network using operator"
+executeAction $testdataDir/basic-network-spec.yml upgradeNetwork
+
+echo "Invoking transactions"
+executeAction $testdataDir/smoke-test-input.yml invoke
+
+echo "Upgrading fabric network using operator"
+executeAction $testdataDir/basic-network-spec.yml updateCapability
```
Comment (scottz64) [inline regression/upgrade/upgrade1.4to2.2.sh]: Upgrading fabric network capabilities using operator

File: regression/upgrade/upgrade1.4to2.2.sh [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+echo "Upgrading fabric network using operator"
```
Comment (scottz64) [inline regression/upgrade/upgrade1.4to2.2.sh]: Upgrading fabric network policies using operator

File: regression/upgrade/upgrade1.4to2.2.sh [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+executeAction $testdataDir/basic-network-spec.yml updatePolicy
+
+echo "Upgrading fabric network using operator"
+executeAction $testdataDir/smoke-network-spec.yml upgradeNetwork
```
Comment (scottz64) [inline regression/upgrade/upgrade1.4to2.2.sh]: Just checking - is this correct to do this twice (but previous time used basic-network-spec.yml)?
You might want to change the echo statement text to differentiate.

File: tools/operator/main.go (hunk 1 above)
Comment (scottz64) [inline tools/operator/main.go]: Failed to upgrade network database

File: regression/upgrade/upgrade1.4to2.2.sh (hunk 4 above)
Comment (suryalnvs) [reply regression/upgrade/upgrade1.4to2.2.sh]: yes, first just upgrading one organization and leaving second organization at previous version. Later for 2 organizations. Fixed in latest update.

**Review by lindluni — APPROVED** [review]

**Pull request merged**